src/styles/_catalog.scss

```diff
@@ -1,10 +1,25 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  justify-items: center;
+  align-items: center;
 
+  grid-template-columns: 1, minmax(200px, 1fr);
```

Comment:
Previous comment not fixed, please move width to variable
![image](https://user-images.githubusercontent.com/54936044/228379991-4c5d1961-cff9-4dae-9650-37f17f05d14e.png)